frame/contracts/rpc/runtime-api/src/lib.rs

```diff
@@ -39,7 +39,7 @@ sp_api::decl_runtime_apis! {
 	{
 		/// Perform a call from a specified account to a given contract.
 		///
-		/// See [`pallet_contracts::Pallet::call`].
+		/// See `pallet_contracts::Pallet::call`.
```

Comment:
@athei here rpc crate doesn't depend on pallet_contracts, so it can't link

Reply:
Ahh I see. Thanks for spotting it.